packages/union-component/src/components/DonationFundDropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { SelectHTMLAttributes, useEffect, useState } from 'react';
+import DonationDropdown from './DonationDropdown';
+import { getMembershipFunds, IFund } from './utils';
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+  setFundId: (fundId: string) => void;
+}
+
+const DonationFundDropdown: React.FC<Props> = (props) => {
```

Comment:
destruct props object `({ setFundId })` I see we only `DonationDropdown` has this _"way of writing"_